policyengine_us/variables/gov/states/az/hhs/tanf/eligibility/az_cash_assistance.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class az_cash_assistance(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Arizona Cash Assistance Payment"
+    definition_period = MONTH
+    reference = (
+        "https://az.db101.org/az/programs/income_support/tanf/program2.htm"
+    )
+    defined_for = StateCode.AZ
+
+    def formula(spm_unit, period, parameters):
+        household_size = spm_unit("spm_unit_size", period)
```

Comment:
need to add a parameter with the value "12" called "max_household_size.yaml - which is the used here as a minimum function to cap the household size at 12